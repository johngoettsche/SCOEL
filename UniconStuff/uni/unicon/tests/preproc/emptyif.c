/* test empty branches */

#if 1
#endif

a

#if 0
#else
#endif

b

#ifdef x
#else
#endif

#ifndef x
#else
#endif

c

#define x 1
#ifdef x
#else
#endif

#ifndef x
#else
#endif

d

#if 0
#elif 0
#elif 1
#elif 0
#endif

e
