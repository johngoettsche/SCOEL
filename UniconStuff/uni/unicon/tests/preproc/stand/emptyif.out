#line 5 "emptyif.c"

a

#line 11 "emptyif.c"

b

#line 21 "emptyif.c"

c

#line 32 "emptyif.c"

d

#line 40 "emptyif.c"

e
